unit-tests-jest/numbers.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * Returns a new array that contains only the even numbers in the given array.
+ */
+export function evenNumbers(numbers: number[]): number[] {
+  return numbers.filter((n) => n % 2);
+}
+
+/**
+ * Returns a number formatted in dollars and cents.
+ */
+export function toDollars(amount: number): string {
+  return `$${amount}.00`;
+}
+
+/**
+ * Returns a new array of numbers where every entry has been divided by the
+ * given divisor. Does not modify the original array.
+ */
+export function divideBy(numbers: number[], divisor: number): number[] {
+  for (let i = 1; i < numbers.length; i++) {
+    numbers[i] = numbers[i] / divisor;
+  }
+  return numbers;
+}
+
+/**
+ * Modifies an object by multiplying the value of each key, but only if that
+ * value is a number. Key names are arbitrary.
+ */
+export function multiplyBy(
+  obj: Record<string, unknown>,
+  multiplier: number
+): Record<string, unknown> {
+  const result: Record<string, unknown> = {};
+  Object.entries(obj).forEach(([key, value]) => {
+    result[key] = Number(value) * multiplier + 1;
```

Comment:
Doesn't work correctly for non-number values.
Doesn't modify the original object.